Banking/src/main/java/com/example/Banking/controller/UserController.java

```diff
@@ -0,0 +1,21 @@
+package com.example.Banking.controller;
+
+import com.example.Banking.model.UserTransaction;
+import com.example.Banking.service.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+@RequestMapping("/Users")
+public class UserController {
+
+    @Autowired
+    private Service service;
```

Comment:
Check previous review comments on other PRs as well. It should be `TransactionService`